docs/source/models/gbd2017/risk_attributable_causes/iron_deficiency/index.rst

```diff
@@ -47,27 +47,76 @@ represents **all** forms of anemia that are attributable to several causes,
 including causes such as hemoglobinopathies and hemolytic anemias that are not 
 causally attributable to iron deficiency. 
 
-Iron Deficiency Risk Factor
-+++++++++++++++++++++++++++
+Anemia Impairment
++++++++++++++++++
 
-The iron deficiency risk factor in GBD 2017 is modeled as a distribution of 
-hemoglobin concentrations. The risk factor is a **continuous measure**.
+The anemia impairment in GBD 2017 represents the total prevalence of anemia due 
+to all causes modeled in GBD (ex: dietary iron deficiency anemia, anemia due to 
+maternal hemorrhage, sickle cell anemia, etc.). Estimating the total prevalence 
+of the anemia impairment for a given population is the first step in modeling 
+anemia in GBD 2017. This is done by fitting a distribution of hemoglobin levels 
+for that population from primary input data. For GBD 2017, an ensemble 
+distribution was used, which was 40% gamma and 60% mirror gumbel. Source code 
+for this process is available `here <https://stash.ihme.washington.edu/projects/MNCH/repos/anemia/browse/model/envelope>` .
```

Comment:
For some reason, several of your underscores got turned into whitespace, so the links aren't working.

Reply:
I don't know why that happened, but it's fixed now. Thanks for flagging this!